client/connector.go

```diff
@@ -198,6 +199,37 @@ func (c *connector) Insert(tableName string, columnNames []string, rows []Row, u
 	return numRows, nil
 }
 
+// hllHash computes hash value from original value
+func hllHash(dataType memCom.DataType, value interface{}) (uint64, error) {
+	var err error
+	value, err = memCom.ConvertValueForType(dataType, value)
+	if err != nil {
+		return 0, err
+	}
+	out := utils.Murmur3Sum64(unsafe.Pointer(&value), memCom.DataTypeBytes(dataType), 0)
+	return out, nil
+}
+
+// computeHLLValue populate hyperloglog value
+func computeHLLValue(dataType memCom.DataType, value interface{}) (uint32, error) {
+	hashed, err := hllHash(dataType, value)
+	if err != nil {
+		return 0, err
+	}
+	p := uint32(14) // max 16
+	group := uint32(hashed & ((1 << p) - 1))
```

Comment:
why not directly do uint32(hashed & ((1 << 14) - 1))

Reply:
i actually moved this function in to utils and created a const for them